orchagent/aclorch.cpp

```diff
@@ -1140,6 +1146,12 @@ bool AclRuleMirror::validateAddMatch(string attr_name, string attr_value)
         return false;
     }
 
+    if (m_tableType == ACL_TABLE_MIRRORV6 && attr_name == MATCH_ETHER_TYPE)
```

Comment:
Isn't this is done by above if() check.  original file Line 1136.

Reply:
Good catch, thanks!